sonic_data_client/transl_data_client.go

```diff
@@ -335,11 +335,22 @@ func addTimer(c *TranslClient, ticker_map map[int][]*ticker_info, cases *[]refle
 }
 
 func TranslSubscribe(gnmiPaths []*gnmipb.Path, stringPaths []string, pathMap map[string]*gnmipb.Path, c *TranslClient, updates_only bool) {
+	sync_done := false
```

Comment:
Could you please explain what is this change and what potential issue it could resolve?

Reply:
Thanks!